features/steps/fhir.py

```diff
@@ -144,7 +145,8 @@ def step_impl(context):
 @given('there is at least 1 entry')
 def step_impl(context):
     resource = context.response.json()
-    entries = resource.get('entry', [])
+    entries = [entry['resource'] for entry in resource.get('entry', [])
```

Comment:
Indeed -- I get `get`'s behavior :-) The scenario I'm concerned about is if we run `GET /AllergyInteolerance?patient=123` and we receive bundle with a single result, an `OperationOutcome` with no `mode`. The issue with relaxed checks here is that we tend never to notice when we've got them wrong (case in point with this issue, which I think has been present for many months but it leads to green check marks in the test suite, which makes everyone happy ;-))